### PR TITLE
Fix LR for LORA and some Adaptation Logic

### PR DESCRIPTION
**Pull request by ArrowM**

### CODE REVIEW

File: dreambooth/optimization.py [hunk 1]
```diff
@@ -512,7 +512,6 @@ def __init__(
             num_cycles: int = 1,
             power: float = 1.0,
             factor: float = 0.5,
-            lr: float = 1e-6,
```
Comment (ArrowM) [inline dreambooth/optimization.py:515]: unused

File: dreambooth/optimization.py [hunk 2]
```diff
@@ -573,65 +572,65 @@ def log_dadapt(disable: bool = True):
     else:
         return 5
 
-def get_optimizer(args, params_to_optimize):
+
+def get_optimizer(optimizer: str, learning_rate: float, weight_decay: float, params_to_optimize):
```
Comment (ArrowM) [inline dreambooth/optimization.py:576]: Now that we have separate fields for Lora LR rates, we need to pass the learning rate rather than passing args and using args.learn_rate in this function.

File: helpers/mytqdm.py [hunk 3]
```diff
@@ -7,6 +7,7 @@
 
 class mytqdm(tqdm):
     def __init__(self, iterable: Iterable = None, **kwargs):
+        self.user = None
```
Comment (ArrowM) [inline helpers/mytqdm.py:10]: I get an error without this

File: dreambooth/train_dreambooth.py [hunk 4]
```diff
@@ -92,13 +92,6 @@ def check_and_patch_scheduler(scheduler_class):
 user_model_dir = ""
 
 
-def dadapt(optimizer):
```
Comment (ArrowM) [inline dreambooth/train_dreambooth.py:95]: I removed this and put `if 'adapt' in args.optimizer:` everywhere
